xcodeutil/testsummaries.go

```diff
@@ -6,9 +6,92 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-tools/go-xcode/plistutil"
 )
 
+// const ...
+const (
+	OldTestSummaries TestSummaryType = "OldTestSummaries"
```

Comment:
Old and New are not the best names, what if later Apple updates the format again? EvenMoreNew...
The old format has Attachments and the new: ScreenshotData.
What do you think about: `TestSummariesWithAttachemnts` and `TestSummariesWithScreenshotData`?